CMakeLists.txt

```diff
@@ -43,9 +43,14 @@ endif()
 # User input options                                                   #
 ########################################################################
 
+
 find_package(PkgConfig REQUIRED)
-set_package_properties(PkgConfig PROPERTIES TYPE RECOMMENDED PURPOSE "Used to detect libxc package")
+set_package_properties(PkgConfig PROPERTIES TYPE RECOMMENDED PURPOSE "Used to detect libxc and libint package")
 pkg_check_modules(LIBXC REQUIRED IMPORTED_TARGET libxc)
+if(NOT TARGET PkgConfig::LIBINT)
+  pkg_check_modules(LIBINT REQUIRED IMPORTED_TARGET libint2>=2.6)
+endif()
```

Comment:
@junghans is that not done here?

Reply:
This was a comment regarding 220090a2dab526738c5cdfbcb8f4bf2e1ab387ed, which changed that. Not sure if you reverted it afterwards.

Reply:
ahh there was some merge problems going on. I saw your version and I liked it.